modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Card.tsx

```diff
@@ -36,6 +42,8 @@ export function Card({
 	viewMode,
 	...otherProps
 }: CardProps) {
+	const [showCardAlert, setShowCardAlert] = useState(true);
```

Comment:
@igor-franca I think you can call this state just `showAlert`. We are not adding the prefix in the other properties like `noCardHeaderBorder` or `cardCustomHeader`.